CHANGELOG.md

```diff
@@ -17,7 +17,8 @@
   - `syscalls::allow_ptr` becomes `syscalls::raw::allow`
 - Targets without support for atomics can be built
 - Most API functions, including `main()`, return a `Result<T, TockError>`
-- The timer now supports to be used simultaneously. To make this safe, `TimerDriver` is now a singleton.
+- The timer now supports to be used simultaneously
```

Comment:
Is this supposed to read something like "The timer now supports parallel alarms"? As-is it's not grammatically correct and hard to understand.

Reply:
Thanks for pointing that out!